### ISSUE TITLE
Testing css

### ISSUE THREAD

**Issue by djmtype** [google/eleventy-high-performance-blog#57]
During development, `npm run watch`, how is one suppose to test css if the code has been minified? There are no line references because everything is on line 1.

**Comment by hhh968574** [comment]
# *@lonekorean ***

**Comment by cramforce** [comment]
The CSS should be unminified. There really is no good reason for the current state since it gets re-minified during build.

**Comment by djmtype** [comment]
@cramforce True, the css isn't minified in the css file, but it does get transformed/minified (probably due to csso) during production – `npm run watch`. Therefore, there is no real frame of reference when troubleshooting.

![Screen Shot 4](https://user-images.githubusercontent.com/930713/100800373-8a48dd80-33f4-11eb-9435-56a3d0751b10.png)

**Comment by cramforce** [comment]
Should be totally possible to generate a sourcemap!

**Comment by djmtype** [comment]
@cramforce  CSSO provides an option, but then the css gets parsed through PurgeCSS anyway, I think.

Please don't keep me in suspense. How do add css source maps for `npm run watch` only?

**Comment by cramforce** [comment]
I filed https://github.com/FullHuman/purgecss/issues/526

There is, indeed, little that can be done for now short of turning off CSS purging during development.

**Comment by theor** [comment]
I've disabled css inlining in dev mode, as it allows browsersync to inject css and bypass 11ty, which is way faster. is it something worth a PR ?

**Comment by cramforce** [comment]
I don't think I want to make that a default, but otherwise it is worth a PR for sure!

**Comment by theor** [comment]
https://github.com/google/eleventy-high-performance-blog/pull/108 let me know what you think !
